velox/exec/Task.cpp

```diff
@@ -1799,6 +1799,12 @@ TaskStats Task::taskStats() const {
     }
   }
 
+  auto bufferManager = bufferManager_.lock();
```

Comment:
I think rlock is sufficient here? thanks!

Reply:
I just realized that bufferManager_ is defined as
`std::weak_ptr<PartitionedOutputBufferManager> bufferManager_;`

So here lock function is not a lock on a mutex, I'm going to change the name `getUtilizationLocked` back to `getUtilization` if no objection?

Reply:
I thought it is a synchronized object.